src/controllers/movies.js

```diff
@@ -0,0 +1,49 @@
+const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+const getMovies = async (req, res) => {
+  const Movies = await prisma.movie.findMany();
```

Comment:
Be careful with capitalisation, `Movies` suggests this is a class name.

Additionally, you've missed some of the core criteria for this function - check the API spec to be sure your response matches the same structure as what's described there. The tests for the core criteria are failing, you can also run `npm test` to see which routes need some work